source/release-notes.txt

```diff
@@ -10,6 +10,29 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.4
+----------------
+
```

Comment:
Note to External Reviewer: We need a release date.

Reply:
March 10, 2025 is the release date